### ISSUE TITLE
Unselected tabs label text have insufficient color contrast ratio (3.36:1)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4568]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
**Test URL** http://cx-guidelines.vmware.com/clarity3FullTesting/
**OS** Tested on Windows but issue is independent of platform.
**User Agents/Assistive Technologies** Firefox 76.0

**Description**
Not selected tabs label text is having insufficient color contrast ratio i.e. it is 3.36:1. Due to this issue, low vision users might find it difficult to read the label of tabs and thus it will be difficult for them to browse through the tabs.

**Steps to Reproduce**
1. Open the test URL.
2. Navigate to the tab section and observe the color contrast ratio of not selected tabs label.

To meet the AA conformance level the color contrast ration should be at least 4.5:1 for a text size less than 14pt bold or 18pt. For text size equal to or greater than 14pt bold or 18pt, ratio color contrast ratio can be at least 3:1.
